mercurial.yaml

```diff
@@ -31,27 +40,48 @@ pipeline:
       uri: https://www.mercurial-scm.org/release/mercurial-${{package.version}}.tar.gz
       expected-sha256: 629604293df2be8171ec856bf4f8b4faa8e4305af13607dce0f89f74132836d6
 
-  - runs: |
-      python3 -m gpep517 build-wheel \
-        --wheel-dir dist \
-        --output-fd 3 3>&1 >&2
-
-      python3 -m installer -d "${{targets.destdir}}" \
-        dist/*.whl
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+        - mercurial
+      runtime:
+        - python-${{range.key}}
+    pipeline:
+      - runs: |
+          # Build the wheel for this python version
+          python${{range.key}} -m gpep517 build-wheel \
+            --wheel-dir dist \
+            --output-fd 3 3>&1 >&2
 
-      install -Dm755 contrib/hg-ssh hgeditor -t "${{targets.destdir}}"/usr/bin
+          # Install the built wheel
+          python${{range.key}} -m installer -d "${{targets.subpkgdir}}" dist/mercurial-6.9-cp*.whl
 
-      for man in doc/*.?; do
-        install -Dm644 "$man" \
-          "${{targets.destdir}}"/usr/share/man/man${man##*.}/${man##*/}
-      done
+          # Copy the hg bins to the right location
+          install -Dm755 contrib/hg-ssh hgeditor -t "${{targets.subpkgdir}}/usr/bin"
 
-  - uses: strip
+          # Cleanup the wheel so the other subpackages don't try to install it too
+          rm -f dist/mercurial-6.9-cp*.whl
```

Comment:
This will result in failure in future when there will be a new version. `mercurial-${{package.version}}-cp*.whl` would be better

Reply:
Oops, of course it would, good catch, thanks!